cpu/atmega_common/periph/spi.c

```diff
@@ -64,7 +68,9 @@ int spi_init_master(spi_t dev, spi_conf_t conf, spi_speed_t speed)
     SPCR = ((1 << SPE) | (1 << MSTR) | conf | (speed & SPEED_MASK));
 
     /* clear interrupt flag */
-    (void)SPSR;
+    (void)SPIE;
```

Comment:
I can left it as before since the manual says that reading the `SPSR` will also clear the interrupt flag, so we can revert this.